devs/docs/basics.rst

```diff
@@ -74,6 +74,28 @@ To get a full list of all available tasks, run::
 
     $ ./gradlew tasks
 
+Such as CrateDB uses the pre-configured bundled version of OpenJDK_, it is
```

Comment:
"Such as?" I'm not sure I understand. Do you mean "since"?

Reply:
yes, "since" makes more sense.